addons/sale_timesheet/views/project_portal_templates.xml

```diff
@@ -18,10 +18,16 @@
 	    </td>
         </xpath>
         <xpath expr="//div[@name='allocated_time']" position="after">
-            <span t-if="task.allow_billable and task.sale_line_id and task.sale_line_id.remaining_hours_available" t-attf-class="{{task.remaining_hours_so &lt; 0 and 'text-danger' or ''}}">
-                <div t-if="is_uom_day">Remaining Days on SO: <span t-esc="timesheets._convert_hours_to_days(task.remaining_hours_so)" t-options='{"widget": "timesheet_uom"}'/></div>
-                <div t-else="">Remaining Hours on SO: <span t-esc="task.remaining_hours_so" t-options='{"widget": "float_time"}'/></div>
-            </span>
+            <div t-attf-class="{{task.remaining_hours_so &lt; 0 and 'text-danger' or ''}}" name="remaining_hours">
```

Comment:
```suggestion
            <div t-att-class="{'text-danger': task.remaining_hours_so &lt; 0}" name="remaining_hours">
```

Reply:
It's nice that you commented because I actually forgot to re-add the t-if on this div :sweat_smile:

Reply:
Also, this suggestion doesn't work because it is a python view, not an owl one :(

Reply:
Looks good! I pushed it :)